src/store/poolFactory/actions.js

```diff
@@ -311,7 +310,7 @@ export const getPoolFactoryFinalLpArray = (finalArray, walletAddress) => async (
           ? null
           : getPoolContract(finalArray[i].poolAddress)
       const synthContract =
-        finalArray[i].synthAddress === false || !walletAddress
+        finalArray[i].synthAddress === false
```

Comment:
This unneeded check was breaking the finalLpArray if the user did not have a connected wallet